### PR TITLE
New package: py-or-tools

### PR DESCRIPTION
**Pull request by aweits**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-or-tools/package.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    depends_on('glog@0.4.0')
+    depends_on('protobuf@3.12.2')
+    depends_on('abseil-cpp@20200225.2')
+    depends_on('cbc@2.10.5')
+    depends_on('cgl@0.60.3')
+    depends_on('clp@1.17.4')
+    depends_on('osi@0.108.6')
+    depends_on('coinutils@2.11.4')
+    depends_on('swig')
+    depends_on('python')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-or-tools/package.py]: Presumably it is also needed at run-time?

File: var/spack/repos/builtin/packages/py-or-tools/package.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+class PyOrTools(CMakePackage):
+    """This project hosts operations research tools developed at
+    Google and made available as open source under the Apache 2.0
+    License."""
+
+    homepage = "https://developers.google.com/optimization/"
+    url      = "https://github.com/google/or-tools/archive/v7.8.tar.gz"
+
+    version('7.8', sha256='d93a9502b18af51902abd130ff5f23768fcf47e266e6d1f34b3586387aa2de68')
+
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-or-tools/package.py:16]:
```
depends_on('cmake@3.14:', type='build')
```

File: var/spack/repos/builtin/packages/py-or-tools/package.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    depends_on('py-six')
+    depends_on('gflags@2.2.2')
+    depends_on('glog@0.4.0')
+    depends_on('protobuf@3.12.2')
+    depends_on('abseil-cpp@20200225.2')
+    depends_on('cbc@2.10.5')
+    depends_on('cgl@0.60.3')
+    depends_on('clp@1.17.4')
+    depends_on('osi@0.108.6')
+    depends_on('coinutils@2.11.4')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-or-tools/package.py:31]: All of these specific versions are required? Or can newer versions also work?

File: var/spack/repos/builtin/packages/py-or-tools/package.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    url      = "https://github.com/google/or-tools/archive/v7.8.tar.gz"
+
+    version('7.8', sha256='d93a9502b18af51902abd130ff5f23768fcf47e266e6d1f34b3586387aa2de68')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-numpy@1.0:', type=('build', 'run'))
+    depends_on('py-packaging@16.0:', type=('build', 'run'))
+    depends_on('py-protobuf@3.12.2')
+    depends_on('protobuf@3.12.2')
+    depends_on('py-six')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-or-tools/package.py]: type=build/run?

File: var/spack/repos/builtin/packages/py-or-tools/package.py (hunk 3 above)
Comment (aweits) [reply var/spack/repos/builtin/packages/py-or-tools/package.py:31]: Possible that newer versions are okay, but I'm working off what they've placed in Dependencies.txt for now.

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**